spec/lib/uuid_mixin_spec.rb

```diff
@@ -0,0 +1,24 @@
+describe UuidMixin do
+  let(:test_class) do
+    Class.new(ActiveRecord::Base) do
+      def self.name; "TestClass"; end
+      self.table_name = "service_templates"
+      include UuidMixin
+    end
+  end
+
+  let(:service_template) { FactoryBot.create(:service_template) }
+  let(:custom_button) { FactoryBot.create(:custom_button, :applies_to_class => "Service") }
+
+  context 'with dup overriden' do
```

Comment:
Are you moving the `dup` logic into `UuidMixin` in this PR?

Reply:
Yeeeeaaaaaah.